.github/workflows/test.yaml

```diff
@@ -0,0 +1,80 @@
+# DO NOT EDIT THIS FILE!
+#
+# It's auto-generated by sonata-project/dev-kit package.
+
+name: Test
+
+on:
+  push:
+    branches:
+      - 3.x
+      - master
```

Comment:
Is this really needed?

Reply:
it would be nice to add where we want to trigger actions on push, because otherwise branches like this one will be double github actions, one for the push and one for the pr

Reply:
we can use defined branches on dev-kit to get this list for each repo